group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupCoordinatorShard.java

```diff
@@ -575,18 +581,32 @@ public CoordinatorResult<OffsetDeleteResponseData, CoordinatorRecord> deleteOffs
     public CoordinatorResult<Void, CoordinatorRecord> cleanupGroupMetadata() {
         long startMs = time.milliseconds();
         List<CoordinatorRecord> records = new ArrayList<>();
+        AtomicInteger deletedClassicGroupCount = new AtomicInteger(0);
```

Comment:
Just to confirm, is the `AtomicInteger` used to match the following forEach loop?

Reply:
If you mean by using it to conform to the lambda expression, yes